tools/mOTUs/motus_merge.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="mereg_mOTUs_tables" name="Merge" version="0.1.0+galaxy0" profile="21.05">
+    <description>Merge mOTUs aboundance or count tables</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="biotools"/>
+    <expand macro="requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+    motus merge
+    #if $input_options.input_source == "files"
+        -i '${','.join([str($file) for $file in $input_options.files])}'
+    #elif $input_options.input_source == "directory"
+        -d '$input_options.directory'
+    #end if
+        #if $add_profiles
+        -a '$add_profiles'
+    #end if
+    -o '$output'
+    #if $biom_format
+        -B
+    #end if
+    ]]></command>
+    <inputs>
+        <conditional name="input_options">
+            <param name="input_source" type="select" label="Input source">
+                <option value="files">Input files</option>
+                <option value="directory">Input directory</option>
+            </param>
+            <when value="files">
+                <param name="files" type="data" format="txt" multiple="true" label="mOTU profiles to merge" />
+            </when>
+            <when value="directory">
+                <param name="directory" type="text" label="Directory containing mOTU profiles" help="Specify the directory path." />
+            </when>
+        </conditional>
```

Comment:
```suggestion
        <param name="files" type="data" format="txt" multiple="true" label="mOTU profiles to merge" />
```

Galaxy has no notion of a directory. A user does not now where on the server the data is and should not now.
Putting in multiple files is the way to go I think. So I think the entire conditional can go.